bin/seth-protogen

```diff
@@ -188,6 +158,24 @@ def fix_import(contents, pkg, sub_dir=False):
     )
 
 
+def protoc(src_dir, base_dir, pkg, language="python"):
+    if language == "python":
```

Comment:
Is there still any python in this repo?

Reply:
Yes, some of the RPC tests use python and require the Seth protos